sdk-webapp/i18n/en.ts

```diff
@@ -686,14 +686,14 @@
     <name>set_as_default</name>
     <message>
         <source>set as default</source>
-        <translation>set as default</translation>
+        <translation>Set As Default</translation>
```

Comment:
Here I was unsure, so I compared it with "Save As" in File menu :) OK, I will use the capitalized As